lib/src/main/java/at/bitfire/vcard4android/ContactReader.kt

```diff
@@ -131,20 +130,20 @@ class ContactReader internal constructor(val vCard: VCard, val downloader: Conta
                     uriToUid(prop.uri)?.let { c.members += it }
 
                 is FormattedName ->
-                    c.displayName = StringUtils.trimToNull(prop.value)
```

Comment:
In case of `trimToNull`, I think it's better to have a function because it's used that often and the replacement is a bit more complex.

Ideally an extension `String.trimToNull()`, define in `Utils.kt` after `String.capitalize()`.